### PR TITLE
[Enhancement] Specify fskey of builtin storage volume

### PR DESCRIPTION
**Pull request by TszKitLo40**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java [hunk 1]
```diff
@@ -383,4 +387,16 @@ private Map<String, String> parseParamsFromConfig() {
...
+
+    private String parseBuiltinFsKeyFromConfig() {
+        switch (Config.cloud_native_storage_type.toLowerCase()) {
+            case "s3":
+                String[] bucketAndPrefix = getBucketAndPrefix();
+                return bucketAndPrefix[0];
+            case "hdfs":
+                return Config.cloud_native_hdfs_url;
+            default:
+                return "";
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java:399]: is it right when type is `azblob`?

Comment (TszKitLo40) [reply fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java:403]: I have added a comment for it.

**Review by wyb — APPROVED** [review]

**Review by kevincai — APPROVED** [review]

**Review by kevincai — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 9 / 10 (90.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/SharedDataStorageVolumeMgr.java|9|10|90.00%|[404]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 9 / 10 (90.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/SharedDataStorageVolumeMgr.java|9|10|90.00%|[404]|

**Review by silverbullet233 — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**